engines/queen/metaengine.cpp

```diff
@@ -72,8 +78,68 @@ class QueenMetaEngine : public AdvancedMetaEngine<Queen::QueenGameDescription> {
 	int getMaximumSaveSlot() const override { return 99; }
 	bool removeSaveState(const char *target, int slot) const override;
 	int getAutosaveSlot() const override { return 99; }
+
+	void languageInput(Common::Keymap *perLanguage,const char *_commandKeys);
+	Common::KeymapArray initKeymaps(const char *target) const override;
```

Comment:
This would now replace the engine-default keymaps listed in MetaEngine::initKeymap.
When doing so, you need to review the default keymaps and consider which ones still should be included. 
I'm specifically concerned about the left and right mouse clicks, which usually include joystick button mapping.